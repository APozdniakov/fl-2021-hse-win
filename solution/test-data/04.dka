states=[q1,q2]
alpha=[a,b,c,d]
initial=[djnd]
accepting=[q2]
trans=[q1>a>q2,q2>b>q1]
