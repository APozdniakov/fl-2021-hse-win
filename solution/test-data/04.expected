Error:
E1: A state 'djnd' is not in the set of states